src/lean_spec/subspecs/forkchoice/helpers.py

```diff
@@ -15,7 +20,7 @@
 def get_fork_choice_head(
     blocks: Dict[Bytes32, Block],
     root: Bytes32,
-    latest_votes: Dict[ValidatorIndex, Checkpoint],
+    latest_votes: Dict[ValidatorIndex, SignedValidatorAttestation],
```

Comment:
Any reason why we need the signed data here? Doesn't look to me we need to re-verify